src/main/drivers/light_ws2811strip.h

```diff
@@ -88,6 +90,10 @@ void setUsedLedCount(unsigned ledCount);
 
 bool isWS2811LedStripReady(void);
 
+#ifdef STM32F4
```

Comment:
There is no need to make the contents of header files conditional - this is normally not done in Betaflight, unless it serves to avoid build problems.